packages/cli/src/actions/env.ts

```diff
@@ -94,12 +95,40 @@ export default defineAction(
       mod.exports[it.name] ||= { app: it.content };
     });
 
+    // Get the version of the @brandingbrand/fsapp
+    // Use require.resolve to support monorepos with the paths set to current working directory
+    const {version} = require(require.resolve('@brandingbrand/fsapp/package.json', {paths: [process.cwd()]}));
+
+    // Coerce the version to a comparable semver version i.e. 12.0.0-alpha.1 -> 12.0.0
+    const coercedVersion = semver.coerce(version);
+
+    // If semver cannot parse the version then we cannot compare versions to be able to
+    // determine where project_env_index.js is located - throw error to user
+    if (!coercedVersion) {
+      throw Error("Type Mismatch: cannot parse @brandingbrand/fsapp version");
+    }
+
     // Resolve the path of the project environment index file from @brandingbrand/fsapp
     // There is a chance this could throw an error, this is fine, still even though we checked the dependencies object already
-    const projectEnvIndexPath = require.resolve(
-      "@brandingbrand/fsapp/src/project_env_index.js",
-      { paths: [process.cwd()] }
-    );
+    let projectEnvIndexPath;
+
+    if (semver.satisfies(coercedVersion, "<11")) {
+      projectEnvIndexPath = require.resolve(
+        "@brandingbrand/fsapp/project_env_index.js",
```

Comment:
@crherman7 when testing, it looks like version 10.x.x.x doesn't have a placeholder file, so this resolutions fails. Error: Cannot find module '@brandingbrand/fsapp/project_env_index.js'.

Reply:
great catch! what an interesting tight coupling.